test/test_extractor.py

```diff
@@ -338,13 +317,9 @@ async def test_extract_file_deb_no_tool(
     ):
         """Test the deb file extraction with no extraction tool"""
 
-        if sys.version_info >= (3, 8):
-            mocker.patch("cve_bin_tool.extractor.aio_inpath", return_value=False)
-        else:
-            mocker.patch(
-                "cve_bin_tool.extractor.aio_inpath",
-                coroutine(mocker.Mock(return_value=False)),
-            )
+
+        mocker.patch("cve_bin_tool.extractor.aio_inpath", return_value=False)
```

Comment:
Just a nitpick but there's no need to add newlines before and after, you didn't in all other places.